easybuild/easyconfigs/m/MRChem/MRChem-1.0.0-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'MRChem'
+version = '1.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://mrchem.readthedocs.io'
+description = """MRChem is a numerical real-space code for molecular electronic
+structure calculations within the self-consistent field (SCF) approximations of
+quantum chemistry: Hartree-Fock and Density Functional Theory."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['https://github.com/MRChemSoft/mrchem/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9cdda4d30b2baabb26400742f78ef8f3fc50a54f5218c8b6071b0cbfbed746c3']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('Eigen', '3.3.7'),
+    ('MRCPP', '1.3.6'),
+    ('Python', '3.8.2'),
+    ('XCFun', '2.1.0')
+]
+
+dependencies = [
+    ('MRCPP', '1.3.6'),
```

Comment:
@stigrj there's no need to include `MRCPP` and `Python` as `builddependencies` if you have them as `dependencies`, so please remove them from the former.

Reply:
Thanks @verdurin!